### ISSUE TITLE
Dependecy of external links

### ISSUE THREAD

**Issue by alair-aurea** [alair-aurea/ansible-scripts#2]
As of now, several tools that are not available on the distro repositories need to be pulled from external links. It would be interesting if these files are stored somewhere, so the playbooks do not break due to changes in the links.

**Comment by ghosgitAurea** [comment]
Can't we put those in files directory?

**Comment by alair-aurea** [comment]
Yes, but should we push it to the github repo? I think a better approach is to have some cloud storage with just the files needed. We should discuss this during the meeting.

**Comment by alair-aurea** [comment]
Ask the link for the ftp so we can test if it is quicker.

**Comment by alair-aurea** [comment]
Add files to ftp://gbuildftp:EALhMiuJ8Ry@gbuild.gdev.com

**Comment by joeleonjr** [comment]
Hi! I'm a security researcher @TruffleSecurity. I found a valid API key leaked in a comment in this Issue. I highly recommend rotating the API key immediately (not just deleting the comment). Here's a blog post I wrote on remediating these types of vulnerabilities: https://trufflesecurity.com/blog/remediate-leaked-api-keys-with-key-rotation.<br><br>If you can't find the leaked API key, you can use the following [TruffleHog](https://github.com/trufflesecurity/trufflehog) command to automatically scan for it:<br>`trufflehog github --repo https://github.com/<your_username>/<your_reponame> --issue-comments --only-verified`<br><br>If it's not clear where/what the security issue is, please let me know. I'd be happy to help. Please note that the API key might be in the past edit of a comment.<br><br><br><br>(cc @alair-esw)
